src/App.js

```diff
@@ -22,54 +27,69 @@ function Logout(props) {
 
 class App extends Component {
   render() {
-    const {user, token} = this.props;
+    const { user, token } = this.props;
     if (!user && token !== null) {
       LocalApi.get("/users/me")
         .then(r => this.props.setUserInfo(r.data))
         .catch(e => console.log(e));
     }
     if (!user) {
       return (
-      <BrowserRouter>
-        <Link to="/">Sign In</Link> | <Link to="/register">Register</Link>
-        <Switch>
-          <Route exact path="/"><SigninForm /></Route>
-          <Route path="/register"><RegisterPage /></Route>
-        </Switch>
-      </BrowserRouter>);
+        <BrowserRouter>
+          <Link to="/">Sign In</Link> | <Link to="/register">Register</Link>
+          <Switch>
+            <Route exact path="/">
+              <SigninForm />
+            </Route>
+            <Route
+              exact
+              path="/register"
+              render={props => {
```

Comment:
We need to pass 'props' here, so that 'history' on Register form won't be undefined.
After fixing this, once users register, they will be redirected to login

Reply:
Sorry, I didn't know that. Thanks!

Reply:
All good! I think we do need to use 'prettier' (or any other extension if prettier is not working for you. I'm happy to install any extension that works for you) to keep our code in the same format... Coz I think I had the <...props> there before, but it might be accidentally deleted for some reasons lol